### PR TITLE
Adds head patting

### PR DESCRIPTION
**Pull request by Robertha89**
Helpfull intent
aim head
click person to pat
Pat pat
:cl: PPI
rscadd: Adds head patting
/:cl:
Re-make of #3511

### CODE REVIEW

**Comment by Gummin** [comment]
:+1:

File: code/modules/mob/living/carbon/carbon.dm [hunk 1]
```diff
@@ -269,14 +269,22 @@ mob/living
...
-					var/mob/living/carbon/human/H = src
-					H.w_uniform.add_fingerprint(M)
-				playsound(get_turf(src), 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
-				M.visible_message( \
-					"\blue [M] gives [src] a [pick("hug","warm embrace")].", \
-					"\blue You hug [src].", \
+				if(M.zone_sel.selecting == "head")
+					playsound(get_turf(src), 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
+					M.visible_message( \
+					"<span class='notice'> [M] pats [src] on the head.</span>", \
```
Comment (TheDZD) [inline code/modules/mob/living/carbon/carbon.dm]: You don't need the extra space after a span class.

File: code/modules/mob/living/carbon/carbon.dm [hunk 2]
```diff
@@ -269,14 +269,22 @@ mob/living
...
+					"<span class='notice'> You pat [src] on the head.</span>", \
 					)
+				else
+					playsound(get_turf(src), 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
```
Comment (TheDZD) [inline code/modules/mob/living/carbon/carbon.dm]: The playsound here and and under the `if` statement can just be moved to before the `if` statement.

File: code/modules/mob/living/carbon/carbon.dm [hunk 3]
```diff
@@ -269,14 +269,22 @@ mob/living
...
+					M.visible_message( \
+					"<span class='notice'> [M] gives [src] a [pick("hug","warm embrace")].</span>", \
+					"<span class='notice'> You hug [src].</span>", \
```
Comment (TheDZD) [inline code/modules/mob/living/carbon/carbon.dm]: Same spacing thing in these two.

**Comment by TheDZD** [comment]
Going to merge this once those small bugbears are dealt with, since the other PR already passed the 24 hour mark, and the majority of people (including the admin team) seem to be in favor of it after the reference was removed.

**Comment by Iamgoofball** [comment]
This adds a shitton of snowflake to human code for no gameplay purpose.

:-1:

**Comment by Spacemanspark** [comment]
:+1: It's the little things that make games fun too.

**Comment by Robertha89** [comment]
Ready for any last checks.

File: code/modules/mob/living/carbon/carbon.dm [hunk 4]
```diff
@@ -263,20 +263,28 @@ mob/living
 				AdjustWeakened(-3)
 				playsound(src.loc, 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
 				if(!player_logged)
-					M.visible_message( \
-						"\blue [M] shakes [src] trying to wake [t_him] up!", \
-						"\blue You shake [src] trying to wake [t_him] up!", \
+					M.visible_message(\
+						"<span class ='notice'> [M] shakes [src] trying to wake [t_him] up!</span>",\
+						"<span class ='notice'> You shake [src] trying to wake [t_him] up!</span>",\
```
Comment (TheDZD) [inline code/modules/mob/living/carbon/carbon.dm]: You still left the extra spaces after the span class in all of these, and now there's an extra space inside of the span class. For example, line 268 should be `"span class='notice'>You shake the [src] trying to wake [t_him] up!</span>"`

**Comment by Robertha89** [comment]
done, and done.

File: code/modules/mob/living/carbon/carbon.dm [hunk 5]
```diff
@@ -263,20 +263,28 @@ mob/living
...
-					var/mob/living/carbon/human/H = src
-					H.w_uniform.add_fingerprint(M)
 				playsound(get_turf(src), 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
-				M.visible_message( \
-					"\blue [M] gives [src] a [pick("hug","warm embrace")].", \
-					"\blue You hug [src].", \
+				if(M.zone_sel.selecting == "head")
+					M.visible_message(\
+					"<span class='notice'> [M] pats [src] on the head.</span>",\
+					"<span class='notice'> You pat [src] on the head.</span>",\
```
Comment (TheDZD) [inline code/modules/mob/living/carbon/carbon.dm]: Extra space after the span one these two lines.

File: code/modules/mob/living/carbon/carbon.dm [hunk 6]
```diff
@@ -263,20 +263,28 @@ mob/living
...
 					)
+				else
+
+					M.visible_message(\
+					"<span class='notice'> [M] gives [src] a [pick("hug","warm embrace")].</span>",\
+					"<span class='notice'> You hug [src].</span>",\
```
Comment (TheDZD) [inline code/modules/mob/living/carbon/carbon.dm]: And these two.

**Comment by Robertha89** [comment]
plz no more spaces i've missed.

**Comment by NTSAM** [comment]
:+1: 
:hand: (This was the closest thing they have for patting.)

**Comment by Dumbdumn5** [comment]
:+1:

**Comment by TheDZD** [comment]
Might want to squash your commits, 4 commits for a few line changes in one file is a bit messy.

**Comment by Robertha89** [comment]
Unable to squash, remaking pull.

**Pull request closed without merging**